CHANGELOG.md

```diff
@@ -1,3 +1,18 @@
+### 0.16.0
+
+- CodeAction to open json schema [#395](https://github.com/redhat-developer/yaml-language-server/pull/395)
```

Comment:
```suggestion
- CodeAction to open json schema from yaml error [#395](https://github.com/redhat-developer/yaml-language-server/pull/395)
```
This is just a nitpick so you don't have to change it, I just think the more context we can give in the changelog the better